health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/data/HealthServicesRepository.kt

```diff
@@ -80,7 +93,8 @@ class HealthServicesRepository @Inject constructor(
 /** Store exercise values in the service state. While the service is connected,
  * the values will persist.**/
 sealed class ServiceState {
-    object Disconnected : ServiceState()
+    data object Disconnected : ServiceState()
```

Comment:
why this change?

Reply:
https://kotlinlang.org/docs/whatsnew19.html#stable-data-objects-for-symmetry-with-data-classes

Reply:
TIL!